src/widget/weffect.cpp

```diff
@@ -44,12 +44,13 @@ void WEffect::effectUpdated() {
         EffectPointer pEffect = m_pEffectSlot->getEffect();
         if (pEffect) {
             const EffectManifest& manifest = pEffect->getManifest();
-            if (! manifest.shortName().isEmpty()) {
+            if (!manifest.shortName().isEmpty()) {
                 name = manifest.shortName();
+                description = manifest.name() + ": " + manifest.description();
```

Comment:
you're right, the english ": " translates in french to " : " (in fact, the first space, before the colon, is a non-breaking space).
But I'm afraid this will be hard in transifex to translate that ax the sapce after the colon will not be visible in the interface and it need some hints to be properly translated :(
@esbrandt what do you think ?